### ISSUE TITLE
CreateArrayLiteral translator bug

### ISSUE THREAD

**Issue by lab313ru** [j4k0xb/View8#8]
In the original repo there was the following declaration of the `CreateArrayLiteral` translator:
```
 "CreateArrayLiteral": lambda obj: f"ACCU = new ConstPool{obj.args[0]}",
```

In one of your commits you have removed `new` from some translators including `CreateArrayLiteral`. This caused incorrect behavior:
1. Constant Pool stores a default array instantiator, for example: `[0, 0, 0, 0, 0]`
2. Using `LdaGlobal` v8 loads this initializer into ACCU and then moves ACCU into one of the R regs
3. Using `StaInArrayLiteral` v8 initializes the array items.

In the original repo such sequences are decompilable into the following:
```
r1 = [0, 0, 0, 0, 0]
r1[0] = 1
r1[1] = 2
r1[2] = 3
r1[3] = 4
r1[4] = 5
some_global = r1
```

In your fork it decompiles in a wrong way:
```
r1 = [0, 0, 0, 0, 0]
[0, 0, 0, 0, 0][0] = 1
[0, 0, 0, 0, 0][1] = 2
[0, 0, 0, 0, 0][2] = 3
[0, 0, 0, 0, 0][3] = 4
[0, 0, 0, 0, 0][4] = 5
some_global = [0, 0, 0, 0, 0]
```

**Comment by j4k0xb** [comment]
In the original repo it decompiled to `r1 = new [0, 0, 0, 0, 0]` which would result in a `Uncaught TypeError: [0, 0, 0, 0, 0] is not a constructor` when run

I removed `new` to make it semantically closer to js but seems like `new` (or any other prefix) also affects how simplifying/inlining works :thinking:

e.g. `r0 = ConstantPool[0]` was treated as constant but `r0 = new ConstantPool[0]` wasn't
https://github.com/j4k0xb/View8/blob/12294193ece092b895438523414fb1fba6238988/Simplify/simplify.py#L29-L41

do you have a suggestion on how to fix both of those issues?

**Comment by lab313ru** [comment]
Yeah, that was also my idea. The `new` in this case can be any random word, which should be removed later. I'll check it tomorrow, thanks.

**Comment by lab313ru** [comment]
This branch should also contain unmutabler:
```
# replace constant regs
        if not re.search(r"^(ACCU|CASE_\d+|[ra]\d+) = ", line):
            line = self.replace_reg_with_constant(line, reg_scope)  # added
            return re.sub(r"^mutable ", "", line)  # added
```
